CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 4.3.0  
+  - Open and close connection for each query #147
```

Comment:
Nit:

Putting #147 inside square brackets makes it a link in the markdown doc (see entry for 4.2.4 for details)

Reply:
@robbavey - Thanks and updated.  We should probably strive for some convention here.